tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -99,11 +118,17 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |Main arg5
        |User arg5
        |""".stripMargin,
-    topLines = Option(4)
+    topLines = Option(4),
+    compat = Map(
```

Comment:
Compat section can be removed here also.

Reply:
It's for `arg5` not `arg4`

Reply:
My brain filtered out the line numbers :sweat: